behave/features/steps/cla_callback_start_callback.py

```diff
@@ -0,0 +1,82 @@
+from behave import *
+from features.constants import CLA_FRONTEND_URL
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.common.exceptions import StaleElementReferenceException
+
+
+@given(u'that I am viewing the case "{case_reference}" which has a callback')
+def step_impl(context, case_reference):
+    # Todo: can be removed when feature branch is rebased with https://dsdmoj.atlassian.net/browse/LGA-1824
+    context.case_reference = case_reference
+    url = f"{CLA_FRONTEND_URL}/call_centre/{case_reference}/"
```

Comment:
I know this will probably come out but just to say, if you click on the link to the case from the callback page - I was taken to /call_centre/{case_reference}/diagnosis but not sure if this is necessary